CHANGES.md

```diff
@@ -1,3 +1,27 @@
+## edge-19.6.2
+
+* CLI
+  * Added the `--linkerd-cni-enabled` flag to the `install` subcommands so that
+    `NET_ADMIN` capability is omitted from the control plane's PSP
+* Controller
+  * Fixed security context values so that injection does not fail on
+    restricted PSPs (thanks @codeman9!)
+  * Fixed webhook failure policy in order to avoid timing issues in HA mode
+  * Introduced control plane's PSP and RBAC resources into Helm templates;
+    these policies are only in effect if the PSP admission controller is
+    enabled
+  * Fixed MWC namespace value so that when installing multiple control planes,
+    there is a unique configuration for each one
+  * Removed `UPDATE` operation from proxy-injector webhook because pod
+    mutations are disallowed during update operations
+* Proxy
+  * The `l5d-override-dst` header is now honored for inbound service profiles
+  * Added the prefix of metric reports in log lines
+  * Fixed metrics so that response errors are properly counted
```

Comment:
TIOLI: what do you think of 
```suggestion
  * Fixed response errors not being properly counted in metrics
```

Reply:
_Fixed a bug where errors were not counted in `response_total` metrics_ ?

Reply:
_Include errors in `response_total` metrics_?

I'd like to avoid "fixed a bug ..." bullets